spec/Section 3 -- Type System.md

```diff
@@ -343,9 +343,17 @@ client-specific primitive for time. Another example of a potentially useful
 custom scalar is `Url`, which serializes as a string, but is guaranteed by
 the server to be a valid URL.
 
+When defining an additional scalar, GraphQL systems should use the `@specified`
+directive to provide an RFC3986-compliant URI pointing to a human-readable
+specification of the data format, serialization, and coercion rules for the
+scalar. For example, a GraphQL system providing a `Date` or `DateTime` scalar
+might link to RFC 3339, or some document defining a reasonable subset of that
+RFC. If a specification URI is present, systems must conform to the described
+rules. Built-in scalar types should not provide a URI.
+
 ```graphql example
-scalar Time
-scalar Url
+scalar Time @specified(by: "https://tools.ietf.org/html/rfc3339")
```

Comment:
This is a bit "misleading" example because RFC3339 is normally used to describe date and time. 
How about `scalar DateTime @specified(by: "https://example.com/myDateTime")` ?

Reply:
Using a non RFC link would also make it more clear that it is totally fine to come up with your scalar spec.

Reply:
I think we all agreed to have a specific spec for datetime since it is based on the rfc but more narrow in its implementation. There should be a specific document outlining the rules for this one.

Reply:
Yeah, I might just take this out as an example until we have something more useful to link to.

Reply:
Actually, if I take it out it gets even more confusing because the other example to talk about in the paragraph above is URL and then it gets really hard to parse whether I'm talking about the `URL` scalar as an example, or the URL that you provide in the example to link to the specification.

I'll rename this to `DateTime` for now, with the expectation that when Andi's restricted datetime scalar format doc gets published we'll change the link to that.